GHermeneus/include/GHermeneus/Machine.h

```diff
@@ -72,14 +73,28 @@ namespace GHermeneus
 
             // Extract the Commands from each line
             std::vector<std::optional<Instruction<SSV_T, T>>> extractedCmds(lines.size());
-            std::transform(std::execution::par_unseq, lines.begin(), lines.end(), extractedCmds.begin(), extractCmd);
-            extractedCmds.erase(std::remove_if(std::execution::par_unseq, extractedCmds.begin(), extractedCmds.end(), [](const auto& cmd){
+            if (parallelExecution)
```

Comment:
codestyle: No single line if statements (always put {} around em)

Reply:
damn I did it again.
I will add a .clang-format so it won't happen again